src/pages/home/report/ReportActionCompose.js

```diff
@@ -518,6 +518,7 @@ class ReportActionCompose extends React.Component {
         const isBlockedFromConcierge = ReportUtils.chatIncludesConcierge(this.props.report) && User.isBlockedFromConcierge(this.props.blockedFromConcierge);
         const inputPlaceholder = this.getInputPlaceholder();
         const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
+        const isUserInteractionDisabled = this.props.reportID === '0';
```

Comment:
Can we do a `disabled` prop and pass it through `ReportFooter` as `shouldDisableCompose`. More explicit and less dependent on the `reportID` would be easier to understand.

Reply:
Amazing thank you!